### ISSUE TITLE
macos_bundle with ios base_sdk

### ISSUE THREAD

**Issue by ajanuar** [bazelbuild/rules_apple#1081]
Is it possible to have `macos_bundle` with base SDK iOS?

EarlGrey 2.0 integration requires to create macos bundle with base sdk iOS.

**Comment by brentleyjones** [comment]
Yes. Here is an incomplete (no tests) and bit-rotted commit that adds it: https://github.com/bazelbuild/rules_apple/commit/558ea6a7fa5f2cb15b6bb4278ab12c46b6d11444

I would review and help shepherd something in, if a PR was opened.

**Comment by ajanuar** [comment]
Hi @brentleyjones, I will take a look. thanks!

**Comment by ajanuar** [comment]
Hi @brentleyjones, I found difficulties to merge your branch to upstream. Can you guide me? Thanks

https://github.com/ajanuar/rules_apple/blob/02606992cc6b1cf85f810500220363b7fba6a77f/apple/internal/partials/framework_import.bzl#L197

vs 

https://github.com/bazelbuild/rules_apple/blob/558ea6a7fa5f2cb15b6bb4278ab12c46b6d11444/apple/internal/partials/framework_import.bzl#L190

**Comment by brentleyjones** [comment]
There has been some refactoring. I'll take a stab at getting it back to the same (no-test) state today.

**Comment by brentleyjones** [comment]
#1085
